packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -143,203 +192,138 @@ contract ControllerHelper is FlashControllerHelper, IERC721Receiver {
     /**
      * @notice receive function to allow ETH transfer to this contract
      */
-    receive() external payable {
-        require(msg.sender == weth || msg.sender == address(controller), "Cannot receive eth");
-    }
+    receive() external payable {}
```

Comment:
Does it make sense to restrict to aave as well and keep the other restrictions?

Reply:
Was thinking about it, IMO this is unnecessary check that just consume more gas every time the contract unwrap to ETH, withdraw collateral